src/Illuminate/Foundation/Application.php

```diff
@@ -1143,6 +1143,16 @@ public function flush()
         parent::flush();
 
         $this->loadedProviders = [];
+        $this->bootingCallbacks = [];
+        $this->bootedCallbacks = [];
+        $this->middlewares = [];
```

Comment:
I tried checking `Illuminate\Foundation\Application` and can't seem to find `middlewares` property. Where does this come from?

Reply:
@crynobone You are right, will file a pr to fix that.